test/react16_suggestors/react_style_maps_updater_test.dart

```diff
@@ -0,0 +1,542 @@
+import 'package:over_react_codemod/src/react16_suggestors/constants.dart';
+import 'package:over_react_codemod/src/react16_suggestors/react_style_maps_updater.dart';
+import 'package:test/test.dart';
+
+import '../util.dart';
+
+main() {
+  group('ReactStyleMapUpdater', () {
+    final testSuggestor = getSuggestorTester(ReactStyleMapsUpdater());
+
+    group('updates correctly when there', () {
+      test('is an empty file', () {
+        testSuggestor(expectedPatchCount: 0, input: '');
+      });
+
+      test('are no matches', () {
+        testSuggestor(
+          expectedPatchCount: 0,
+          input: '''
+            main() {
+              Foo()
+              ..id = 'number1'
+              ..style = {
+                'width': 40
+              };
+            }
+          ''',
+        );
+      });
+
+      test('is a single value within a style map', () {
+        testSuggestor(
+          expectedPatchCount: 2,
+          input: '''
+            main() {
+              Foo()
+              ..id = 'number1'
+              ..style = {
+                'width': '40'
+              };
+            }
+          ''',
+          expectedOutput: '''
+            main() {
+              Foo()
+              ..id = 'number1'
+              ${getCheckboxComment(keysOfModdedValues: ['width'])}
```

Comment:
Why is a comment added here? Manual validation shouldn't be required for values we update automatically.